src/components/pages/Categories/Manage.vue

```diff
@@ -48,6 +49,21 @@ export default {
   created() {
     this.$store.dispatch('categories/getCategoryList');
   },
+  methods: {
+    updateValue($event) {
+      const categoryName = $event.target.value;
+      this.category = categoryName;
```

Comment:
[imo]

一時変数になると思うので、以下のような記述で良いかと思います。

```suggestion
      this.category = $event.target.value;
```

Reply:
変更しました！